autoload/lsp/lsp.vim

```diff
@@ -335,7 +335,14 @@ def LspSavedFile(bnr: number)
   endif
 
   for lspserver in lspservers
-    lspserver.didSaveFile(bnr)
+    # TODO: implement `catch` block
+    # Wrap method with `try-finally` block to solve error: 
+    # `E716: Key not present in Dictionary: "supportsDidSave"`
+    try
+      lspserver.didSaveFile(bnr)
+    finally
+      return
```

Comment:
even it happened, just `return` or should `continue`?

Reply:
I didn't dive into the entire code base, I just hid the errors that appeared during programming. Perhaps it would be worthwhile to implement the ability to hide errors using a variable.

Reply:
though @yegappan accepted your/this PR, but i think:
1. this didnot fix (or hid) the real err, since it perhaps was an init case, tho not sure how come that happened.
2. and perhaps worse, since if that really happened, i thought it should `continue` to next lspserver vs `return`.